### PR TITLE
Fix: Handle URLs with path / parameters (#203)

### PR DESCRIPTION
**Pull request by LucasBPM**
Added url parsing to extract hostname from url

### CODE REVIEW

File: modules/src/ping.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 def process(input, entities):
     output = {}
     try:
         url = entities['url'][0]['value']
-        strip_list = ['http://', 'https://', '/']
-        for item in strip_list:
-            url = url.strip(item)
+        if '//' not in url:
+          url = '%s%s'%('//', url)
+        url = urlparse(url).hostname
```
Comment (swapagarwal) [inline modules/src/ping.py]: `hostname` returns `None` if it is unable to parse the url. See https://docs.python.org/2/library/urlparse.html Throw an error in that case.

File: modules/src/ping.py [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+from urlparse       import urlparse
 
 def process(input, entities):
     output = {}
     try:
         url = entities['url'][0]['value']
-        strip_list = ['http://', 'https://', '/']
-        for item in strip_list:
-            url = url.strip(item)
+        if '//' not in url:
```
Comment (swapagarwal) [inline modules/src/ping.py]: This does not look elegant. Can be made better by trying to get the scheme of the parsed url, and adding it if does not exist; then parse it again.

File: modules/src/ping.py [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+          url = '%s%s'%('//', url)
```
Comment (swapagarwal) [inline modules/src/ping.py]: Use 4 spaces for indentation.

**Comment by LucasBPM** [comment]
Not too sure if I handled the exception properly but the rest of it should be good.

**Comment by swapagarwal** [comment]
Thanks @LucasBPM! 👍

**Pull request merged**